latrine_django/latrineapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0.3 on 2018-05-30 22:24
```

Comment:
are these generated files going to be generated at runtime or do they need to be stored in the repository?

Reply:
They need to be stored in the repo. Here is a quote from the Django documentation:

> The migration files for each app live in a “migrations” directory inside of that app, and are designed to be committed to, and distributed as part of, its codebase. You should be making them once on your development machine and then running the same migrations on your colleagues’ machines, your staging machines, and eventually your production machines.
https://docs.djangoproject.com/en/2.0/topics/migrations/